### ISSUE TITLE
Contrast issues discovered in new design

### ISSUE THREAD

**Issue by hexylena** [galaxyproject/galaxy#7550]
We should be meeting WCAG for contast, but labels fail AA, much less AAA.

![image](https://user-images.githubusercontent.com/458683/54543511-ae9d2600-4995-11e9-91c6-c3a1d3550fe1.png)

It should be `#494949` at least to pass AAA.

**Comment by wm75** [comment]
Maybe a more severe contrast issue is inside *Data Libraries*:
While the Data Libraries overview page itself looks fine, there seems to be a stylesheet issue when you step inside any folder (low contrast, different font).

**Comment by hexylena** [comment]
The history dropdown menu probably also has this issue. There we could probably switch to bold/black.

![image](https://user-images.githubusercontent.com/458683/54679815-3010c780-4b00-11e9-922f-2cc57c29c7b2.png)

**Comment by mblue9** [comment]
The pop-up boxes for the interactive tours could also use some more contrast if possible to make them easier to spot.

**Comment by hexylena** [comment]
![image](https://user-images.githubusercontent.com/458683/57648151-038ba000-75c5-11e9-9e62-c286241d5108.png)

Does not have enough contrast that I can recognise that 'edit' or 'run' are links

**Comment by hexylena** [comment]
![image](https://user-images.githubusercontent.com/458683/59279735-c4547b80-8c64-11e9-84af-7112d3fff57f.png)

is still a pretty bad issue on 19.05, as @wm75 reported

**Comment by martenson** [comment]
@erasche It does not seem that bad on pure 19.05, maybe eu's local styles are affecting it?

![Galaxy___Data_Libraries](https://user-images.githubusercontent.com/1814954/59280532-bb978180-8c33-11e9-8dbe-891501a221af.png)

**Comment by hexylena** [comment]
Odd, our styling overrides are really simple. I'll look into it more. Only change from us is `#masthead { background-color: #003399; }` appended to end of css file

**Comment by hexylena** [comment]
![image](https://user-images.githubusercontent.com/458683/59282017-848f9300-8c68-11e9-8669-ae63748a4f61.png)

**Comment by hexylena** [comment]
Removing a

```
font-weight: lighter
```

seems to fix it.

**Comment by martenson** [comment]
@erasche 'lighter' shouldn't mean color change, it should be changing boldness/weight. Yours is clearly lighter in terms of color. What combination of OS&browser is it?

**Comment by hexylena** [comment]
linux + chrome / firefox. Is this a "was designed for retina displays" type issue?

With `font-weight: normal`

![image](https://user-images.githubusercontent.com/458683/59282398-26af7b00-8c69-11e9-9bb7-3990742ddd52.png)

**Comment by martenson** [comment]
@erasche what do you see here? https://developer.mozilla.org/en-US/docs/Web/CSS/font-weight#Result

**Comment by hexylena** [comment]
hah! was just making screenshots of that page

![image](https://user-images.githubusercontent.com/458683/59282525-59597380-8c69-11e9-8a96-f75dc480d17d.png)

![image](https://user-images.githubusercontent.com/458683/59282546-61b1ae80-8c69-11e9-948f-50b0a61d9801.png)

that demo you linked to does parent: bold, and then a "lighter" of that parent which I think resets it to normal weight? But for the above examples, yeah, it's too light especially at small font sizes.

**Comment by martenson** [comment]
@erasche I am happy to change it. It seems to be warned about on the MDN page too.

**Comment by hexylena** [comment]
Ok, sounds good. Sorry weird issue that we see different things.

**Comment by martenson** [comment]
the tool panel on dev has proper contrast on `dev`
![Galaxy](https://user-images.githubusercontent.com/1814954/59288624-35366c00-8c42-11e9-8263-a1667624f780.png)

library list has normal weight font (400) on dev
![Galaxy___Data_Libraries](https://user-images.githubusercontent.com/1814954/59288665-497a6900-8c42-11e9-9a26-28639fb4f96a.png)

**Comment by martenson** [comment]
after https://github.com/galaxyproject/galaxy/pull/8133/commits/482eb71525c544d5eb30bd1d5623328288e19688
![Galaxy___Tours](https://user-images.githubusercontent.com/1814954/59289205-7da25980-8c43-11e9-8f47-8dc178976428.png)

**Comment by martenson** [comment]
after https://github.com/galaxyproject/galaxy/pull/8133/commits/b0761c3f7bd9e9717ed7bfa0229a0dc2edd5a3ee
![Galaxy](https://user-images.githubusercontent.com/1814954/59290115-d1ae3d80-8c45-11e9-9b63-47517b36bb18.png)
![Galaxy](https://user-images.githubusercontent.com/1814954/59290124-d70b8800-8c45-11e9-81cb-dc2f335dc587.png)

**Comment by martenson** [comment]
after https://github.com/galaxyproject/galaxy/pull/8133/commits/c9a262da36ba2323bbeb7740bb773c488a5164b7
![Galaxy](https://user-images.githubusercontent.com/1814954/59290788-5a79a900-8c47-11e9-8e4d-eba6d38d7512.png)

**Comment by hexylena** [comment]
Thanks for commit numbers, we'll backport them.

**Comment by martenson** [comment]
I think with https://github.com/galaxyproject/galaxy/pull/8133 merged we have addressed everything in this issue, closing now. Thanks for the insights and feel free to open a new one when you have more feedback!

**Comment by hexylena** [comment]
Awesome! :D
